From 72ec6adb2f4f899680e29c05fc1e2346a3eb4a3e Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Mon, 9 Jan 2023 14:21:16 +0100
Subject: [PATCH 0784/1695] plat-stm32mp1: conf: enable ADC support on
 stm32mp13 platforms

Add STM32 ADC support to STM32MP13 platform.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git d5052816c68fff6dbffc880c97c0b113b2820ae8 plat-stm32mp1: conf: enable ADC support on stm32mp13 platforms]
Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Change-Id: Ib8668373f8d0cf87a07bd2f1b80b3532b1480315
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/372785
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/plat-stm32mp1/conf.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index d24ee5ad8..b9e2dc8b5 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -168,9 +168,11 @@ $(call force,CFG_BOOT_SECONDARY_REQUEST,n)
 $(call force,CFG_CORE_ASYNC_NOTIF,y)
 $(call force,CFG_CORE_ASYNC_NOTIF_GIC_INTID,31)
 $(call force,CFG_CORE_RESERVED_SHM,n)
+$(call force,CFG_DRIVERS_ADC,y)
 $(call force,CFG_DRIVERS_CLK_FIXED,y)
 $(call force,CFG_SCMI_MSG_PERF_DOMAIN,y)
 $(call force,CFG_SECONDARY_INIT_CNTFRQ,n)
+$(call force,CFG_STM32_ADC,y)
 $(call force,CFG_STM32_EXTI,y)
 $(call force,CFG_STM32_GPIO,y)
 $(call force,CFG_STM32_HSE_MONITORING,y)
-- 
2.47.2

